components/ItemTag.tsx

```diff
@@ -0,0 +1,129 @@
+import { Button, Image, Text, VStack } from '@chakra-ui/react';
+import { useMemo } from 'react';
+
+import { PlayerActions, useActions } from '@/contexts/ActionsContext';
+import { useGame } from '@/contexts/GameContext';
+import { Item } from '@/utils/types';
+
+const fontSizeMap = {
+  sm: 'xs',
+  md: 'sm',
+  lg: 'md',
+};
+
+const smallFontSizeMap = {
+  sm: '2xs',
+  md: 'xs',
+  lg: 'sm',
+};
+
+const widthMap = {
+  sm: '6rem',
+  md: '8rem',
+  lg: '10rem',
+};
+
+const paddingMap = {
+  sm: 2,
+  md: 4,
+  lg: 6,
+};
+
+const spacingMap = {
+  sm: 2,
+  md: 3,
+  lg: 4,
+};
+
+type Size = 'sm' | 'md' | 'lg';
+
+type ItemTagProps = {
+  holderId?: string;
+  item: Item;
+  size?: Size;
+};
+
+export const ItemTag: React.FC<ItemTagProps> = ({
+  size = 'md',
+  item,
+  holderId,
+}) => {
+  const { fontSize, smallFontSize, width, padding, spacing } = useMemo(
+    () => ({
+      fontSize: fontSizeMap[size],
+      smallFontSize: smallFontSizeMap[size],
+      width: widthMap[size],
+      padding: paddingMap[size],
+      spacing: spacingMap[size],
+    }),
+    [size],
+  );
+
+  const { character } = useGame();
+
+  const { itemId, name, image, amount } = item;
+
+  const showActions = useMemo(() => {
+    return (
+      size != 'sm' &&
+      !!holderId &&
+      !!character &&
+      holderId === character.characterId
+    );
+  }, [size, character, holderId]);
+
+  const isEquipped = useMemo(() => {
+    return (
+      !!holderId &&
+      !!character &&
+      holderId === character.characterId &&
+      character.equippedItems.find(h => h.itemId === itemId)
+    );
+  }, [character, holderId, itemId]);
+
+  const { openActionModal, selectCharacter, selectItem } = useActions();
+
+  return (
+    <VStack
+      p={padding}
+      spacing={spacing}
+      w={width}
+      bg={isEquipped ? 'black' : 'white'}
```

Comment:
Looks like you can't actually see the button when it's unequipped:

<img width="348" alt="Screenshot 2023-09-13 at 1 06 03 PM" src="https://github.com/raid-guild/character-sheets-frontend/assets/40322776/9c91e6cb-a2ce-44cd-a6d5-f716d4e5b6ba">

Reply:
Oh, I think I reviewed this locally too early... may just give it a gold look or something, for kicks

Reply:
Tweaked in 2418bbff836c46514d33244e1d9aba49ea79cd5a